test/configCases/runtime/depend-on-diamond-chain/webpack.config.js

```
@@ -0,0 +1,32 @@
+"use strict";
+
+// A chain of `dependOn` diamonds: root -> [left, right] -> join -> [left, right]
+// -> join ... Every join is reachable by two paths, so a traversal that does not
+// remember visited entrypoints costs 2 ** DEPTH visits of the last one. `node`
+// chunk loading is what pulls in `getRuntimeChunkDependentChunksIterable`
+// (StartupChunkDependenciesPlugin requires `ensureChunkIncludeEntries`).
+// At this depth the undeduplicated traversal needs ~50s, so it fails the test
+// timeout rather than merely being slow.
+const DEPTH = 16;
+
+/** @type {NonNullable<import("../../../../").Configuration["entry"]> & object} */
```

ESLint

[error] Please use provide types for object  - `{ property: number:, result: () => number}` instead `Object`/`object` or use `EXPECTED_OBJECT` type

(jsdoc/no-restricted-syntax)